plugins/woocommerce/tests/e2e-pw/tests/merchant/create-cart-block.spec.js

```diff
@@ -33,8 +32,7 @@ test.describe(
 			await closeChoosePatternModal( { page } );
 
 			await fillPageTitle( page, testPage.title );
-			const wordPressVersion = await getInstalledWordPressVersion();
-			await insertBlock( page, 'Classic Cart', wordPressVersion );
+			await insertBlock( page, 'Classic Cart', Date.now().toString() );
```

Comment:
The test fails on external envs because of `getInstalledWordPressVersion()`. For this reason, we skipped all those tests to run on external envs. Here, instead of getting the WP version, I'm using a timestamp because I don't see any other usage of the WP version.